contracts/minters/vending-minter-wl-flex/src/contract.rs

```diff
@@ -730,9 +730,15 @@ fn _execute_mint(
         .add_attribute("sender", info.sender)
         .add_attribute("recipient", recipient_addr)
         .add_attribute("token_id", mintable_token_mapping.token_id.to_string())
-        .add_attribute("network_fee", network_fee)
-        .add_attribute("mint_price", mint_price.amount)
-        .add_attribute("seller_amount", seller_amount))
+        .add_attribute(
+            "network_fee",
+            coin(network_fee.u128(), mint_price.clone().denom).to_string(),
+        )
+        .add_attribute("mint_price", mint_price.to_string())
```

Comment:
this looks good although not sure if to_string is necessary

Reply:
Receiving a `the trait From<Coin> is not implemented for std::string::String` error otherwise.